spec/datetimeformat.html

```diff
@@ -500,7 +504,10 @@
           The array that is the value of the "nu" property of any locale property of [[LocaleData]] must not include the values "native", "traditio", or "finance".
         </li>
         <li>
-          [[LocaleData]][locale] must have hourCycle property with String values for all locale values.
+          [[LocaleData]][locale].hc must be the array [*null*, *"h11"*, *"h12"*, *"h23"*, or *"h24"*] for all locale values.
```

Comment:
ok, a list makes more sense. thanks @anba, I have added an issue to track that here https://github.com/tc39/ecma402/issues/156